test/src/util.js

```diff
@@ -18,6 +18,9 @@ export function uniqueId() {
   return Date.now().toString();
 }
 
+export const TEST_USER_FIRST_NAME = 'Test';
+export const TEST_USER_LAST_NAME = `User_${uniqueId()}`;
```

Comment:
You might consider adding a `TEST_USER_INITIALS` so you don't have to sprinkle `${TEST_USER_FIRST_NAME.charAt(0)}${TEST_USER_LAST_NAME.charAt(0)}` everywhere

Reply:
Ah good catch, I'll add that in #985